test/unit/transforms/util/pullRequestLinkGenerator.test.ts

```diff
@@ -0,0 +1,33 @@
+import { generateCreatePullRequestUrl } from "../../../../src/transforms/util/pullRequestLinkGenerator";
+
+describe("pullRequestLinkGenerator()", () => {
+
+	it("should structure the url correctly with issue keys in title - for single issue key", async () => {
+		const BASE_URL = "github";
+		const NAME = "branch-7";
+		const ISSUE_KEYS = ["TEST-222"];
+		const ISSUE_KEYS_FLAT = "TEST-222"
+		const EXPECTED_RESULT = `github/compare/${NAME}?title=${ISSUE_KEYS_FLAT}%20-%20${NAME}`;
```

Comment:
Let's use the exact string in the Expected Result. 

Otherwise this test repeats the implementation